### PR TITLE
[UP-453] Example remove unused resources

### PR DESCRIPTION
**Pull request by kellemNegasi**
Provides a solution for [UP-453](https://unidoc.atlassian.net/browse/UP-453)

### CODE REVIEW

File: pages/pdf-split-with-unused-resources.go
Comment (sampila) [inline pages/pdf-split-with-unused-resources.go:1]: change the file name to `pdf_split_and_removes_unused_resources.go`

File: pages/pdf-split-with-unused-resources.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/*
+ * This example shows how to remove unused resource from pdf pages while splitting.
+ *
+ * Run as: go run pdf-split-with-unused-resources.go <input.pdf> <output-dir>
```
Comment (sampila) [inline pages/pdf-split-with-unused-resources.go:4]: Run as: go run pdf_split_and_removes_unused_resources.go <input.pdf> <output-dir>

File: pages/pdf-split-with-unused-resources.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
+/*
+ * This example shows how to remove unused resource from pdf pages while splitting.
```
Comment (sampila) [inline pages/pdf-split-with-unused-resources.go:2]: This example shows how to split pdf pages and removes unused resources.

**Review by sampila — COMMENTED** [review]
Great, added comments

**Review by sampila — APPROVED** [review]
Looks good to me

**Review by gunnsth — APPROVED** [review]

**Pull request merged**